### ISSUE TITLE
Error from gorm on registration with production Postgres

### ISSUE THREAD

**Issue by remexre** [project-domino/project-domino#20]
```
2016-06-20T08:01:27.045215+00:00 app[web.1]: 2016/06/20 08:01:27 [Recovery] panic recovered:
2016-06-20T08:01:27.045216+00:00 app[web.1]: POST /register HTTP/1.1
2016-06-20T08:01:27.045217+00:00 app[web.1]: Host: project-domino-testing.herokuapp.com
2016-06-20T08:01:27.045217+00:00 app[web.1]: Connection: close
2016-06-20T08:01:27.045218+00:00 app[web.1]: Accept: text/plain, */*; q=0.01
2016-06-20T08:01:27.045219+00:00 app[web.1]: Accept-Encoding: gzip, deflate, br
2016-06-20T08:01:27.045219+00:00 app[web.1]: Accept-Language: en-US,en;q=0.8,zh-CN;q=0.6,zh;q=0.4,ko;q=0.2
2016-06-20T08:01:27.045220+00:00 app[web.1]: Connect-Time: 1
2016-06-20T08:01:27.045220+00:00 app[web.1]: Connection: close
2016-06-20T08:01:27.045221+00:00 app[web.1]: Content-Type: application/x-www-form-urlencoded; charset=UTF-8
2016-06-20T08:01:27.045222+00:00 app[web.1]: Cookie: auth=
2016-06-20T08:01:27.045222+00:00 app[web.1]: Dnt: 1
2016-06-20T08:01:27.045223+00:00 app[web.1]: Origin: https://project-domino-testing.herokuapp.com
2016-06-20T08:01:27.045224+00:00 app[web.1]: Referer: https://project-domino-testing.herokuapp.com/register
2016-06-20T08:01:27.045224+00:00 app[web.1]: Total-Route-Time: 0
2016-06-20T08:01:27.045225+00:00 app[web.1]: User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.84 Safari/537.36
2016-06-20T08:01:27.045226+00:00 app[web.1]: Via: 1.1 vegur
2016-06-20T08:01:27.045226+00:00 app[web.1]: X-Forwarded-For: 174.53.253.80
2016-06-20T08:01:27.045227+00:00 app[web.1]: X-Forwarded-Port: 443
2016-06-20T08:01:27.045228+00:00 app[web.1]: X-Forwarded-Proto: https
2016-06-20T08:01:27.045228+00:00 app[web.1]: X-Request-Id: b02972b9-dc0d-4c2d-a32d-92ad18185153
2016-06-20T08:01:27.045229+00:00 app[web.1]: X-Request-Start: 1466409686914
2016-06-20T08:01:27.045229+00:00 app[web.1]: X-Requested-With: XMLHttpRequest
2016-06-20T08:01:27.045230+00:00 app[web.1]: 
2016-06-20T08:01:27.045230+00:00 app[web.1]: 
2016-06-20T08:01:27.045231+00:00 app[web.1]: reflect.Value.Addr of unaddressable value
2016-06-20T08:01:27.045231+00:00 app[web.1]: /app/tmp/cache/go/src/runtime/panic.go:443 (0x437cc9)
2016-06-20T08:01:27.045232+00:00 app[web.1]: /app/tmp/cache/go/src/reflect/value.go:240 (0x6a42fc)
2016-06-20T08:01:27.045233+00:00 app[web.1]: /app/tmp/cache/gopath/src/github.com/jinzhu/gorm/callback_create.go:120 (0x4dc3da)
2016-06-20T08:01:27.045234+00:00 app[web.1]: /app/tmp/cache/gopath/src/github.com/jinzhu/gorm/scope.go:798 (0x50cd58)
2016-06-20T08:01:27.045234+00:00 app[web.1]: /app/tmp/cache/gopath/src/github.com/jinzhu/gorm/main.go:372 (0x4fa95a)
2016-06-20T08:01:27.045235+00:00 app[web.1]: /app/tmp/cache/gopath/src/github.com/project-domino/project-domino/handlers/api/auth.go:107 (0x4b9766)
2016-06-20T08:01:27.045236+00:00 app[web.1]: /app/tmp/cache/gopath/src/github.com/gin-gonic/gin/context.go:97 (0x4bce0a)
2016-06-20T08:01:27.045237+00:00 app[web.1]: /app/tmp/cache/gopath/src/github.com/project-domino/project-domino/middleware/login_middleware.go:59 (0x4bbdc9)
2016-06-20T08:01:27.045237+00:00 app[web.1]: /app/tmp/cache/gopath/src/github.com/project-domino/project-domino/middleware/login_middleware.go:44 (0x4bb70e)
2016-06-20T08:01:27.045238+00:00 app[web.1]: /app/tmp/cache/gopath/src/github.com/gin-gonic/gin/context.go:97 (0x4bce0a)
2016-06-20T08:01:27.045239+00:00 app[web.1]: /app/tmp/cache/gopath/src/github.com/project-domino/project-domino/middleware/database_middleware.go:13 (0x4bc16b)
2016-06-20T08:01:27.045239+00:00 app[web.1]: /app/tmp/cache/gopath/src/github.com/gin-gonic/gin/context.go:97 (0x4bce0a)
2016-06-20T08:01:27.045240+00:00 app[web.1]: /app/tmp/cache/gopath/src/github.com/gin-gonic/gin/recovery.go:45 (0x4cf591)
2016-06-20T08:01:27.045241+00:00 app[web.1]: /app/tmp/cache/gopath/src/github.com/gin-gonic/gin/context.go:97 (0x4bce0a)
2016-06-20T08:01:27.045241+00:00 app[web.1]: /app/tmp/cache/gopath/src/github.com/gin-gonic/gin/logger.go:63 (0x4ce82a)
2016-06-20T08:01:27.045242+00:00 app[web.1]: /app/tmp/cache/gopath/src/github.com/gin-gonic/gin/context.go:97 (0x4bce0a)
2016-06-20T08:01:27.045243+00:00 app[web.1]: /app/tmp/cache/gopath/src/github.com/gin-gonic/gin/gin.go:284 (0x4c3b52)
2016-06-20T08:01:27.045243+00:00 app[web.1]: /app/tmp/cache/gopath/src/github.com/gin-gonic/gin/gin.go:265 (0x4c3787)
2016-06-20T08:01:27.045244+00:00 app[web.1]: /app/tmp/cache/go/src/net/http/server.go:2081 (0x71ec6e)
2016-06-20T08:01:27.045245+00:00 app[web.1]: /app/tmp/cache/go/src/net/http/server.go:1472 (0x71b51e)
2016-06-20T08:01:27.045245+00:00 app[web.1]: /app/tmp/cache/go/src/runtime/asm_amd64.s:1998 (0x468b41)
2016-06-20T08:01:27.045246+00:00 app[web.1]: 
2016-06-20T08:01:27.045251+00:00 app[web.1]: [GIN] 2016/06/20 - 08:01:27 | 500 |   129.39003ms | 174.53.253.80 |   POST    /register
```

**Comment by remexre** [comment]
Just noting, this is _not_ a case of normal error handling being left to a `panic()` call -- this is Gorm attempting an illegal operation via the reflect library.

**Comment by akshay288** [comment]
Resolved by 408e1ce.
